src/e2e-app/BUILD.bazel

```diff
@@ -23,9 +24,18 @@ ng_module(
     "//src/material-examples:examples",
     "//src/cdk-experimental/dialog",
     "//src/cdk-experimental/scrolling",
+    "//src/material-experimental",
```

Comment:
Out of curiosity: Why add this here? 

> We should only add deps that are actively used right?

Reply:
whoops, split this out from a different PR and forgot to remove that